app/src/main/res/layout/lessons_chapter_view.xml

```diff
@@ -42,6 +42,7 @@
       android:fontFamily="sans-serif"
       android:text="@{String.format(@string/topic_play_chapter_index, (viewModel.index + 1))}"
       android:textColor="@{viewModel.chapterPlayState != ChapterPlayState.NOT_PLAYABLE_MISSING_PREREQUISITES ? @color/oppiaPrimaryText : @color/oppiaPrimaryText30}"
+      android:importantForAccessibility="@{viewModel.chapterPlayState != ChapterPlayState.NOT_PLAYABLE_MISSING_PREREQUISITES ? 1 : 2}"
```

Comment:
What does `1` and `2` mean here? Also, I think it should be replaced with something which makes sense instead of `1` and `2`

Reply:
Actually ` android:importantForAccessibility` attribute accepts yes or no as a value, whose integer equivalent is 1 and 2 respectively.

Reply:
![Screenshot 2021-01-24 194125](https://user-images.githubusercontent.com/64064110/105633164-1586a280-5e7d-11eb-8969-d6c1753c0e0a.png)

Reply:
As per the discussion, we will be using enumeration in place of 1 and 2